tests/mpls/test_mpls.py

```diff
@@ -0,0 +1,401 @@
+import os
+import time
+import random
+import logging
+import pprint
+import json
+
+from abc import ABCMeta, abstractmethod
+
+import pytest
+
+import ptf.testutils as testutils
+import ptf.mask as mask
+import ptf.packet as packet
+
+from tests.common import reboot, port_toggle
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+from tests.common.fixtures.duthost_utils import backup_and_restore_config_db_module
+from tests.common.config_reload import config_reload
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.disable_loganalyzer,  # disable automatic loganalyzer
+    pytest.mark.topology('t1')
+]
+BASE_DIR = os.path.dirname(os.path.realpath(__file__))
+DUT_TMP_DIR='/tmp'
+ADD_DIR = os.path.join(BASE_DIR, 'configs')
+
+LABEL_POP_ROUTES='label_pop_routes.json'
+LABEL_PUSH_ROUTES='label_push_routes.json'
+LABEL_SWAP_ROUTES='label_swap_routes.json'
+LABEL_DEL_ROUTES='label_del_routes.json'
+
+@pytest.fixture(scope='module')
+def setup(duthost, tbinfo, ptfadapter):
+    """
+    setup fixture gathers all test required information from DUT facts and tbinfo
+    :param duthost: DUT host object
+    :param tbinfo: fixture provides information about testbed
+    :return: dictionary with all test required information
+    """
+    if tbinfo['topo']['name'] not in ('t1'):
+        pytest.skip('Unsupported topology')
```

Comment:
This check is not required because the script already has mark:
```
pytestmark = [
    pytest.mark.disable_loganalyzer,  # disable automatic loganalyzer
    pytest.mark.topology('t1')
]
```

Reply:
Removed check.